Aplib.Core/Tactics/AnyOfTactic.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Aplib.Core.Tactics
+{
+    /// <summary>
+    /// Represents a tactic that executes any of the provided sub-tactics.
+    /// </summary>
+    public class AnyOfTactic : Tactic
+    {
+        /// <summary>
+        /// Gets or sets the sub-tactics of the tactic.
+        /// </summary>
+        protected LinkedList<Tactic> SubTactics { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AnyOfTactic"/> class with the specified sub-tactics.
+        /// </summary>
+        /// <param name="subTactics">The list of sub-tactics.</param>
+        public AnyOfTactic(List<Tactic> subTactics)
```

Comment:
What is the added value of using a List as opposed to an array? Arrays also allow params.
```suggestion
        public AnyOfTactic(params Tactic[] subTactics)
```